chalicelib/checks/wrangler_checks.py

```diff
@@ -2438,7 +2454,7 @@ def find_lab(record, all_labs, all_grants):
         # matcher = stringmatch.Levenshtein()
         for disp in all_lab_names:
             # s = round(matcher.get_sim_score(record['OH Lab'], disp.split(',')[0]) * 100)
-            s = fuzz.token_sort_ratio(record['OH Lab'], disp.split(',')[0])
+            score = round(string_label_similarity(record['OH Lab'], disp.split(',')[0]) * 100)
```

Comment:
You changed "s =" to "score =". Is that intended? Looks like a bug.

Reply:
Great catch - thanks!